README.md

```diff
@@ -1,8 +1,55 @@
 # Northcoders News API
 
+## Summary
+
+Northcoders News API is my *first* backend development project. The intention here is to mimic the construction of a real-world backend service (e.g. Reddit), which provides application data to the front end architecture.
+
+The database is PSQL, and interactions with the database will be handled via node-postgres.
+
+## Live Demo
+
+You can access the live demo of the project [here](postgres://khgpmtkp:rPedU8wcCotDKKIZzOG79Y2nfk7BuKpj@trumpet.db.elephantsql.com/khgpmtkp).
```

Comment:
That link doesn't work. You need to ensure it is hosted correctly. 
Have another look through these instructions https://notes.northcoders.com/courses/js-back-end/api-hosting